hippylib/modeling/cpp_AssemblePointwiseObservation/AssemblePointwiseObservation.cpp

```diff
@@ -41,7 +47,7 @@ PetscInt PointwiseObservation::computeLGtargets(MPI_Comm comm,
 	 std::vector<int> tmp(nTargets);
 	 for(int i = 0; i < nTargets; ++i)
 	 {
-		 Point p(gdim, targets.data()+i*gdim);
+		 dolfin::Point p(gdim, targets.data()+i*gdim);
 		 if(bbt->collides_entity(p))
```

Comment:
~~Isn't `collides_entity(p)` deprecated and removed starting from 2018.1.0?~~ Apparently it is only removed from python API